meson.build

```diff
@@ -48,15 +48,25 @@ add_project_arguments(
 
 # Configuration
 
-if get_option('large-tiles')
-    conf.set('CONFIG_LARGE_TILES', 1)
-endif
-
-conf.set('PACKAGE_NAME', 'libass')
-conf.set('PACKAGE_VERSION', meson.project_version())
+conf.set('CONFIG_LARGE_TILES', get_option('large-tiles').to_int())
 
 conf.set('CONFIG_SOURCEVERSION', '"meson, commit: @VCS_TAG@"')
 
+p_name = '"@0@"'.format(meson.project_name())
+p_vers = '"@0@"'.format(meson.project_version())
+
+conf.set('PACKAGE', p_name)
+conf.set('PACKAGE_BUGREPORT', '""')
+conf.set('PACKAGE_NAME', p_name)
+conf.set(
+    'PACKAGE_STRING',
+    '"@0@ @1@"'.format(meson.project_name(), meson.project_version()),
+)
+conf.set('PACKAGE_TARNAME', p_name)
+conf.set('PACKAGE_URL', '""')
+conf.set('PACKAGE_VERSION', p_vers)
+conf.set('VERSION', p_vers)
+
```

Comment:
nice catch with `CONFIG_UNIBREAK` and `LARGE_TILES` *(the latter has no practical effect atm but it definitely should match)*; thanks!

I’m not convinced replicating currently unused autoconf macros is actually beneficial though. If we never use them it’ſ just harmless cruft. If for whatever reason we start using more of those (and then presumably also adapt `configure.ac` to yield different values rather than empty strings or duplicates of other macros) there are two possibilities:
- we remember to also change meson and all is fine
- we forget changing the meson definition
  - if currently unused macros are undefined in meson, this will make CI fail, we’ll notice and fix it
  - if they are already defined with old or stub values, CI will pass, we won’t notice and meson builds now yield different info than expected